### DISCUSSION TITLE
Possibility of co hosting

### DISCUSSION THREAD

**Discussion by coff33ninja** [AzuraCast/AzuraCast#5859] [Ideas]
If possible a setup of web dj with co hosting support without the need of using third party tools like Radioboss.

**Comment by Mozitux** [comment]
By co-hosting, do you mean being able to host a live show with several participants directly from the webdj page?

**Reply by coff33ninja** [reply]
Yes it would give more livelihood to the stream if others where to be able to logon and co stream like on the radios

**Reply by Mozitux** [reply]
Unfortunately, this is not possible in the current state. The only solution is to go through a third-party service that you can self-host. Eventually, you can suggest this feature, but don't wait for it to come out soon. :/
